### PR TITLE
feat(edit-contentlet): Allow WYSIWYG user setting systemwide TinyMCE props

### LINKED ISSUE #28097: Edit Contentlet: Allow WYSIWYG user setting systemwide TinyMCE props
### Parent Issue

https://github.com/dotCMS/core/issues/25445

### Task

- [x] Support all the field variables for the WYSIWYG field. [docs](https://www.dotcms.com/docs/latest/the-wysiwyg-field#TinyMCEProps)

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### Acceptance Criteria

Uses should be able to configure the WYSIWYG field through a JSON file.

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by rjvelazco**
### Proposed Changes
-  Use systemwide `tinyMCE` props in the new `WYSIWYS` field

### Checklist
- [x] Tests
- [x] Security Implications Contemplated (add notes if applicable)

### Video

https://github.com/dotCMS/core/assets/72418962/b5edc845-f391-4069-a410-1890a1705659

### CODE REVIEW

**Review by oidacra — APPROVED** [review]

**Review by KevinDavilaDotCMS — APPROVED** [review]

File: core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html [hunk 1]
```diff
@@ -1 +1,3 @@
+@if (init()) {
 <editor [formControlName]="field.variable" [init]="init()" />
```
Comment (fmontes) [inline core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html:2]: `async` pipe better?

**Review by fmontes — CHANGES_REQUESTED** [review]

File: core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html (hunk 1 above)
Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html:2]: The `init()` is a signal, not an observable

Comment (rjvelazco) [reply core-web/libs/edit-content/src/lib/fields/dot-edit-content-wysiwyg-field/dot-edit-content-wysiwyg-field.component.html:2]: I can make it an `observable` if you want

File: core-web/libs/data-access/src/lib/dot-scripting-api/dot-scripting-api.service.spec.ts
[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/data-access/src/lib/dot-scripting-api/dot-scripting-api.service.spec.ts:1]: I'll make the request in the component itself

[the remark this answers is not in the document]
Comment (rjvelazco) [reply core-web/libs/data-access/src/lib/dot-scripting-api/dot-scripting-api.service.spec.ts:1]: Done!

**Review by hmoreras — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**